docs/synapse/userguides/model_updates/update_v2_197_0.rst

```diff
@@ -0,0 +1,63 @@
+
+
+.. _userguide_model_v2_197_0:
+
+######################
+v2.197.0 Model Updates
+######################
+
+The following model updates were made during the ``v2.197.0`` Synapse release.
+
+*********
+New Forms
+*********
+
+``doc:requirement``
+  A single requirement, often defined by a standard.
+
+
+``doc:resume``
+  A cv/resume document.
```

Comment:
```suggestion
  A CV/resume document.
```

Reply:
This will also require updating the model docstring.

Reply:
Seems like CV should be capitalized since it's an initialism.